docs/moduleAna3AIMEC.rst

```diff
@@ -7,6 +7,10 @@ For this purpose, avalanche simulation results are transformed into an avalanche
 (see :numref:`fig-aimec-comp-real`, :numref:`fig-aimec-comp-new`, :numref:`fig-aimec-domain-transfo`),
 which enables the comparison of different simulations (for example varying parameter sets,
 or performed with different models) of the same avalanche catchment, in a standardized way.
+There is also the option to compare simulation results to a reference data set, which can consist of a reference
+point, line or polygon. The simulation results are analysed regarding a chosen result variable (peak pressure, thickness or
+velocity) and a threshold of this variable and then compared towards the reference point, line or polygon.
```

Comment:
```suggestion
velocity), a threshold of this variable and then compared towards the reference point, line or polygon.
```

Reply:
I think in this case the second and doesn't belong to the listing of inputs but just adds the next step in the procedure 
so I rephrased it: "The simulation results are analysed regarding a chosen result variable (peak pressure, thickness or
velocity) and a threshold of this variable. The derived runout point and/or line is then compared to the reference point, line or polygon."